css/style.css

```diff
@@ -0,0 +1,101 @@
+#main-doc {
+	font-family: "Open Sans", Arial;
+	font-size: 0.96em;
+  position: absolute;
+  margin-left: 310px;
+  padding: 20px;
+  margin-bottom: 110px;
+}
+
+header {
+	font-size: 1.8em;
+	color: black;
+	display: block;
+}
+
+.main-section {
+	color: #4d4e53;
+}
+
+p {
+	line-height: 24px;
+	margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+	display: block;
+}
+
+div {
+    display: block;
+    list-style-type: disc;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    padding-inline-start: 20px;
+}
+
+section li {
+	margin: 15px 0px 0px 20px;
+}
+
+code {
+	background-color: rgb(247, 247, 247);
+	color: rgb(77, 78, 83);
+	display: block;
+	font-family: monospace;
+	font-size: 12px;
+	line-height: 25px;
+	margin: 10px;
+	padding: 15px;
+	border-radius: 5px;
+}
```

Comment:
Please, I kindly suggest there should be a maximum indentation of two spaces in your file to make it look clean and easy to read. 👍